main.go

```diff
@@ -18,7 +19,8 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/pelletier/go-toml"
+
+
```

Comment:
Now we have a bunch of empty lines in the import block - `gofmt -s` gets rid of more than one newline.
Note that I'm talking about the [`gofmt`](https://golang.org/cmd/gofmt/) tool here, not the `go fmt` subcommand.

Reply:
ahh, just updated